### PR TITLE
Allow running latest OSS-Fuzz corpora across all OSS-Fuzz benchmarks.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/fuzzbench/issues/162

### CODE REVIEW

**Comment by inferno-chromium** [comment]
PTAL.

File: experiment/run_experiment.py [hunk 1]
```diff
@@ -57,6 +58,9 @@
                                  r'^third_party/oss-fuzz/build/|'
                                  r'^docker/generated.mk$|'
                                  r'^docs/)')
+OSS_FUZZ_CORPUS_BACKUP_URL_FORMAT = (
```
Comment (lszekeres) [inline experiment/run_experiment.py:61]: nit: does this need to be a global? if yes, does it need to be public (ie without the _ prefix)?

**Review by lszekeres — APPROVED** [review]

File: experiment/test_run_experiment.py [hunk 2]
```diff
@@ -199,19 +199,30 @@ def test_copy_resources_to_bucket(tmp_path):
...
-                run_experiment.copy_resources_to_bucket(config_dir, config)
-                mocked_cp.assert_called_once_with(
-                    'src.tar.gz',
-                    'gs://gsutil-bucket/experiment/input/',
-                    parallel=True)
-                mocked_rsync.assert_called_once_with(
-                    'config',
-                    'gs://gsutil-bucket/experiment/input/config',
-                    parallel=True)
+        with mock.patch('common.filestore_utils.cp') as mocked_filestore_cp:
```
Comment (jonathanmetzman) [inline experiment/test_run_experiment.py:207]: It's surprising to me that you want to mock both the filestore and gsutil versions.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: experiment/run_experiment.py (hunk 1 above)
Comment (inferno-chromium) [reply experiment/run_experiment.py:61]: Constants we usually try to put in global all over code. Added _ prefix.

**Pull request merged**